storage/cache/flag.go

```diff
@@ -7,11 +7,26 @@ import (
 	flipt "github.com/markphelps/flipt/rpc"
 	"github.com/markphelps/flipt/storage"
 	"github.com/pkg/errors"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 	"github.com/sirupsen/logrus"
 )
 
 const flagCachePrefix = "flag:"
 
+var (
+	cacheHitTotal = promauto.NewCounter(prometheus.CounterOpts{
+		Name:        "cacheHitTotal",
```

Comment:
could you make the metric names use snake_case so they match the existing metrics that prometheus provides plz?

Reply:
Looks like replace all got the better of me! Thank you for pointing it out, I’ll be sure to convert back to snake case.